lib/screens/transfer/send/send_confirmation/interactor/mappers/send_transaction_state_mapper.dart

```diff
@@ -22,9 +22,20 @@ class SendTransactionStateMapper extends StateMapper {
     } else {
       final resultResponse = result.asValue!.value as SendTransactionResponse;
 
+      final int currentDate = DateTime.now().millisecondsSinceEpoch;
+      bool _shouldShowInAppReview = shouldShowInAppReview;
+
+      if (settingsStorage.dateSinceLastAsk != null && shouldShowInAppReview) {
+        final int millisecondsPerMoth = 24 * 60 * 60 * 1000 * 30;
+        final dateUntilAppRateCanAsk = settingsStorage.dateSinceLastAsk! + millisecondsPerMoth;
```

Comment:
`dateSinceLastAsk` - the first time we run the app this value is not there? Will this line crash?

Reply:
it can be null so no crash.